src/templates/token.tsx

```diff
@@ -196,11 +310,67 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
               <MjmlText
                 mjClass="font-accent text-secondary"
                 fontSize={values.px16}
-                fontWeight={700}
+                lineHeight={values.px18}
+                fontWeight={400}
               >
                 <span className="break-all">{secret}</span>
               </MjmlText>
-              <MjmlSpacer height={values.px20} />
+              <MjmlText
+                fontSize={values.px16}
+                color={colors.formal}
+                mjClass="font-primary"
+              >
+                <a
+                  style={gmailLinkStyles}
+                  href={`https://sealc.red/app?domain=${domain}&token=${secret}`}
+                >
+                  <img
+                    src={`${assetsEndpoint}/token_button.png`}
+                    className="hover-img-button img-button"
+                    style={{ verticalAlign: 'middle', ...gmailLinkStyles }}
+                  />
+                </a>
+              </MjmlText>
+              <MjmlSpacer height={values.px6} />
+              <MjmlDivider
+                borderWidth={values.px}
+                borderColor={colors.darkBlue}
+              />
+              <MjmlSpacer height={values.px32} />
+              <MjmlWrapper
+                backgroundColor={colors.primaryBackground}
+                borderRadius={values.px8}
+                verticalAlign="middle"
+              >
+                <MjmlSpacer height={values.px12} />
+                <MjmlText
+                  mjClass="font-accent text-secondary"
+                  fontWeight={700}
+                  align={aligns.center}
+                  fontSize={values.px14}
+                  lineHeight={values.px22}
+                >
+                  <span>
+                    This token/url/button is a password, never share it with
+                    anyone!
+                  </span>
+                </MjmlText>
+                <MjmlSpacer height={values.px12} />
+              </MjmlWrapper>
+              <MjmlSpacer height={values.px24} />
+              <MjmlText
+                mjClass="font-accent text-formal-accent"
+                fontWeight={400}
+                fontSize={values.px14}
+                lineHeight={values.px22}
+              >
+                <span>
+                  If button doesn’t work, copy and paste this token url into
+                  your browser:{' '}
+                  {`https://sealc.red/app?domain=${domain}&token=${secret}`}
```

Comment:
You have this in 2 places, let's extract it into `const`

Reply:
I have extracted it.